src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,658 @@
+
+Create an external secret that specifies the LMS certificate, key, and (optional) CA certificate.
+
+For example, the following `ExternalSecret` called `lms-tls` uses AWS Secrets Manager as the [`SecretStore`](https://external-secrets.io/latest/introduction/getting-started/#create-your-first-secretstore), and references a remote [AWS secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/create_secret.html) called `lms-certs`:
+
+~~~ yaml
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: lms-tls
+spec:
+  refreshInterval: 1h
+  secretStoreRef:
+    name: aws-secret-store
+    kind: SecretStore
+  target:
+    name: lms-tls
+    creationPolicy: Owner
+    template:
+      engineVersion: v2
+      data:
+        lms-ca.crt: '{% raw %}{{ .caCert }}{% endraw %}'
+        lms-tls.crt: '{% raw %}{{ .serverCert }}{% endraw %}'
+        lms-tls.key: '{% raw %}{{ .serverKey }}{% endraw %}'
+  data:
+  - secretKey: caCert
+    remoteRef:
+      key: lms-certs
+      property: caCert
+  - secretKey: serverCert
+    remoteRef:
+      key: lms-certs
+      property: serverCert
+  - secretKey: serverKey
+    remoteRef:
+      key: lms-certs
+      property: serverKey
+~~~
+
+Each `.crt` and `.key` file is associated with its corresponding property in the remote secret `lms-certs`.
+
+In the [Helm configuration](#configuration), `lms.sslVolumes` and `lms.sslVolumeMounts` specify [volumes](https://kubernetes.io/docs/concepts/storage/volumes/#secret) and mount paths that contain server-side certificates for the LMS. The path to each file is specified as an environment variable in `lms.env`. Cockroach Labs recommends mounting certificates to `/app/certs`.
+
+~~~ yaml
+lms:
+  sslVolumes:
+    - name: lms-tls
+      secret:
+        secretName: lms-tls
+  sslVolumeMounts:
+    - mountPath: "/app/certs"
+      name: lms-tls
+      readOnly: true
+  env:
+    - name: LMS_SSL_CA
+      value: /app/certs/lms-ca.crt
+    - name: LMS_SSL_CERT
+      value: /app/certs/lms-tls.crt
+    - name: LMS_SSL_KEY
+      value: /app/certs/lms-tls.key
+~~~
+
+#### Configure orchestrator and client certificates
+
+Create an external secret that specifies the orchestrator certificate, key, and (optional) CA certificate.
+
+For example, the following `ExternalSecret` called `orch-tls` uses AWS Secrets Manager as the [`SecretStore`](https://external-secrets.io/latest/introduction/getting-started/#create-your-first-secretstore), and references a remote [AWS secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/create_secret.html) called `orch-certs`:
+
+~~~ yaml
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: orch-tls
+spec:
+  refreshInterval: 1h
+  secretStoreRef:
+    name: aws-secret-store
+    kind: SecretStore
+  target:
+    name: orch-tls
+    creationPolicy: Owner
+    template:
+      engineVersion: v2
+      data:
+        orch-ca.crt: '{% raw %}{{ .caCert }}{% endraw %}'
```

Comment:
@Jeremyyang920 @ryanluu12345 I'm just guessing that these need to be the actual file names of the certs. Let me know if that's wrong.